templates/zerver/api/upload-image.md

```diff
@@ -0,0 +1,40 @@
+# Upload an image
+
+Upload a single image file and get the corresponding URI.
+
+You may only upload one file at a time.
+
+`POST {{ api_url}}/v1/user_uploads`
+
+## Usage examples
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+</ul>
+<div class="blocks">
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|upload-image|example}
+
+</div>
+
+</div>
+
+</div>
+
+## Arguments
+
+{generate_api_arguments_table|arguments.json|upload-image.md}
+
+## Response
+
+#### Return values
```

Comment:
Probably somewhere on this page we should mention that there's a file size limit, which is configurable by the administrator of the Zulip server via `MAX_FILE_UPLOAD_SIZE`, which defaults to 25MB.